examples/demo/pingpong/src/pingpong/j/model/Player.java

```diff
@@ -24,7 +23,6 @@
 	public class BallAtPlayerPort extends Port<BallIfc, BallIfc> {
 	}
 
-	@BehaviorPort
```

Comment:
Why was this necessary?

Reply:
Because it is not a behavior port (out port cannot be behavior port anyway) but it is a delegation port to an lower level component. In case of delegation connection the parent port should not be behavior port so the generated cpp code does not complies in this case.